package.json

```diff
@@ -13,7 +13,12 @@
     "engines": {
         "vscode": "^1.74.0"
     },
-    "categories": [],
+    "categories": [
+        "Debuggers",
```

Comment:
not sure if its appropriate for us

Reply:
you can run tests, right?

Reply:
for me it sounds like debugging similar to what we have in IDE :)

Reply:
we don't have "runners" category, thus (I assume) debugging means: run, troubleshoot, fix, run again, etc.